src/theme/components.tsx

```diff
@@ -115,12 +115,15 @@ export const StyledInternalLink = styled(Link)`
   }
 `
 
+export const ExternalLinkNoLineHeight = styled(ExternalLink)`
+  line-height: 0;
+`
+
 const StyledLink = styled.a`
   text-decoration: none;
   cursor: pointer;
   color: ${({ theme }) => theme.primary};
   font-weight: 500;
-  line-height: 0;
```

Comment:
why remove this

Reply:
<img width="399" alt="image" src="https://user-images.githubusercontent.com/33005392/183326594-8482aea7-1c19-4982-a7ea-0b418b27c668.png">
revert code.

Reply:
✅